addons/project/tests/test_project_ui.py

```diff
@@ -12,15 +11,19 @@ def test_01_project_tour(self):
 
     def test_project_task_history(self):
         """This tour will check that the history works properly."""
-        project = self.env['project.project'].create({
+        stage = self.env['project.task.type'].create({'name': 'To Do'})
+        _, project2 = self.env['project.project'].create([{
```

Comment:
In odoo, it is a bit bad practise to name our variable `_` because the translation function is named `_` and we could override it. 

Fortunately, we are in a test, so it could be acceptable but if we don't do it here, we will not think to that in the business code. 😇 

Let's do rename the variable into `_dummy` for instance. 🙂 

```suggestion
        _dummy, project2 = self.env['project.project'].create([{
```

Reply:
Yeah, forget about that, done